besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -536,6 +537,13 @@ private InetAddress autoDiscoverDefaultIP() {
               + " (default: ${DEFAULT-VALUE})")
   private final NetworkName network = null;
 
+  @Option(
+      names = {PROFILE_OPTION_NAME},
+      paramLabel = PROFILE_FORMAT_HELP,
+      description =
+          "Overwrite default settings. Possible values are ${COMPLETION-CANDIDATES}. (default: none)")
```

Comment:
Not sure what this should say exactly... Just created #6430 to add something in the help footer so it shows at the bottom, so I think we can refine this later. 

Choose a profile to run Besu with that set of defaults. For more info see...